ors-test-scenarios/src/test/java/integrationtests/ConfigEnvironmentTest.java

```diff
@@ -192,5 +194,35 @@ void testActivateEachProfileWithEnvAndOverwriteDefaultConfig(ContainerInitialize
             OrsApiHelper.assertProfilesLoaded(container, allProfiles.stream().collect(HashMap::new, (m, v) -> m.put(v, true), HashMap::putAll));
             container.stop();
         }
+
+        /**
+         * arg-overrides-default-prop.sh
+         */
+        @MethodSource("utils.ContainerInitializer#ContainerTestImageDefaultsImageStream")
+        @ParameterizedTest(name = "{0}")
+        @Execution(ExecutionMode.CONCURRENT)
+        void testNonExistentConfigFail(ContainerTestImageDefaults targetImage) {
+            GenericContainer<?> container = initContainer(targetImage, false, "testNonExistentConfigFail");
+
+            // Prepare the environment
+            container.addEnv("ORS_CONFIG_LOCATION", "nonexistent/ors-config.yaml");
+            container.addEnv("ors.config.location", "");
+
+            try {
+                container.start();
+
+                String logs = container.getLogs();
+                Assertions.assertTrue(logs.contains("ORS config file not found at: nonexistent/ors-config.yaml"));
+
+            } catch (ContainerLaunchException e) {
+
```

Comment:
If this catch-block is empty on purpose, it should contain a comment explaining why.